tests/baselines/reference/mappedTypeWithAsClauseAndLateBoundProperty2.errors.txt

```diff
@@ -0,0 +1,11 @@
+tests/cases/compiler/mappedTypeWithAsClauseAndLateBoundProperty2.ts(1,14): error TS4118: The type of this node cannot be serialized because its property '__@iterator@8' cannot be serialized.
+tests/cases/compiler/mappedTypeWithAsClauseAndLateBoundProperty2.ts(1,14): error TS4118: The type of this node cannot be serialized because its property '__@unscopables@10' cannot be serialized.
+
+
+==== tests/cases/compiler/mappedTypeWithAsClauseAndLateBoundProperty2.ts (2 errors) ====
+    export const thing = (null as any as { [K in keyof number[] as Exclude<K, "length">]: (number[])[K] });
+                 ~~~~~
+!!! error TS4118: The type of this node cannot be serialized because its property '__@iterator@8' cannot be serialized.

```

Comment:
Ack, this error leaks the internal symbol name structure. I'd prefer if we could print almost anything else (after all, it's not going to be clear to a user what `__@iterator@8` is) - `symbolToString`'s imprecise result `[iterator]` is probably better than `symbolName` here.